src/components/Card.js

```diff
@@ -35,15 +44,30 @@ export default class Card {
   }
 
   _handleLikeIcon() {
-    this._cardElement
-      .querySelector(".card__like-icon")
-      .classList.toggle("card__like-icon_active");
+    if (this.isLiked) {
+      this._cardElement
+        .querySelector(".card__like-icon")
+        .classList.add("card__like-icon_active");
+    } else {
+      this._cardElement
+        .querySelector(".card__like-icon")
+        .classList.remove("card__like-icon_active");
+    }
+  }
+
+  showLikes(isLiked) {
+    this.isLiked = isLiked;
+    this._handleLikeIcon();
   }
 
   _handleOpenDisplay() {
     this._handleCardClick(this._link, this._name);
   }
 
+  deleteCardElement() {
+    this._cardElement.remove();
```

Comment:
The best thing to do after deleting a card is to **remove the link** to the DOM element: `this._cardElement= null;`. It helps javascript garbage collector.
More info: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Memory_Management